msal/tests/Test.MSAL.NET.UIAutomation/MSALMobileTestHelper.cs

```diff
@@ -67,12 +67,12 @@ public static void AcquireTokenSilentTestHelper(ITestController controller, User
 
         private static void AcquireTokenInteractiveHelper(ITestController controller, UserQueryParameters userParams)
         {
-            var user = prepareForAuthentication(controller, userParams);
+            var user = PrepareForAuthentication(controller, userParams);
```

Comment:
Thank you for your attention to detail and cleaning up previous code in this PR.  Adding license files, fixing spelling errors, properly PasalCasing methods, etc.  It's a great habit to have in making the code better when you leave.